src/crossfilter.js

```diff
@@ -121,6 +138,11 @@ function crossfilter() {
     offset = tmp.offset;
     one = tmp.one;
     zero = ~one;
+      
+    // Expose the dimension mask for internal use
+    dimension._offset = offset;
```

Comment:
This is tough. I'm with @esjewett that we should not expose internal variables just to make them accessible within crossfilter.

Another possible approach is to expose an identifier for the dimensions (`offset` is a unique integer I think?), and then keep an array of zeroes and ones. 

IOW add a method `dimension.id()` which we say is unique for all current dimensions. Secretly it is actually `offset`.

I don't understand this:
> And since the offset/one/zero is defined in the dimension constructor itself it would also mean that map in the crossfilter scope would need to be exposed.